src/main/java/org/pih/petl/api/EtlService.java

```diff
@@ -114,37 +130,11 @@ public JobExecution saveJobExecution(JobExecution jobExecution) {
     public void markHungJobsAsRun() {
         for (JobExecution jobExecution : jobExecutionRepository.findJobExecutionsByCompletedIsNullAndStartedIsNotNull()) {
             jobExecution.setCompleted(new Date());
+            jobExecution.setStatus(JobExecutionStatus.ABORTED);
             jobExecutionRepository.save(jobExecution);
         }
     }
 
```

Comment:
I moved job execution out of the service and into the JobExecutor, as there were a lot of circular references and I wanted to keep the concerns separate.